repository-data/application/src/main/resources/hcm-config/configuration/users/sitewriter.yaml

```diff
@@ -0,0 +1,8 @@
+definitions:
+  config:
+    /hippo:configuration/hippo:users/sitewriter:
```

Comment:
@thojava - I assume `sitewriter` user is meant for moderation ? Perhaps we should add the procedure to create `sitewriter` user in the HEE admin documentation (in case if any already). Thoughts ?

Reply:
@smanivasag The `sitewriter` is the system user which used by the delivery tier so it would not be used for moderation. 

@duncanheron @MattBrooksManifesto Do we have idea on how the moderation process for blog comment could be in this moment ?

Reply:
Ok, thanks @thojava 👍